src/pages/publicAppeal/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import Widget from 'src/shared/Widget'
+import { Space, Typography } from 'antd'
+
+import './PublicAppeal.scss'
+const { Title } = Typography
+const pageName = 'publicAppealPage'
+const PublicAppeal = () => {
+  const { t } = useTranslation()
+  const tasks = t(`${pageName}.tasks`, { returnObjects: true })
+
+  return (
+    <PublicAppealStyle>
+      <Space direction="vertical" size="middle">
+        <Title className="page-title" level={3}>
+          {t(`${pageName}.title`)}
+        </Title>
+        {tasks.map((task, i) => (
+          <Task task={task} index={i} key={i} />
+        ))}
+      </Space>
+    </PublicAppealStyle>
+  )
+}
+
+const Task = (props: any) => {
```

Comment:
we don't like `any` in this project - you can use something like
```
type TaskDetails {
    title: string
    swagger: string
}
```

Additionally, I think we only need one swagger link to all sections. It can be nice to have the ability to send `{ childrens }` to this component